.travis.yml

```diff
@@ -11,8 +11,10 @@ addons:
       - ghc
       - zsh
       - ksh
+      - nodejs
```

Comment:
um, why would we need travis to install node from apt? it already has it from nvm. can't the script run `nvm install node`?